.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,58 +1,38 @@
-Hello team,
+|Related issue|
+|---|
+||
 
-`<YOUR_INTRODUCTION_TEXT>`
-
-Closes #`<YOUR_ISSUE_NUMBER>`
-
-# Tests logic
+<!--
+This template reflects sections that must be included in new Pull requests.
+Contributions from the community are really appreciated. If this is the case, please add the
+"contribution" to properly track the Pull Request.
```

Comment:
the "contribution" label?

Reply:
This [one](https://github.com/wazuh/wazuh-qa/labels/community).